pyfarm/agent/etc/agent.yml

```diff
@@ -130,6 +130,13 @@ agent_http_retry_delay_factor: 5
 # be False.
 agent_http_persistent_connections: True
 
+# When using persistent connections, twisted will sometimes run requests over
+# connections that have already been closed by the server without realizing it.
+# When this happens, we catch the resulting failure and retry the request.
+# We do this up to $agent_broken_connection_max_retry times, assuming that if
```

Comment:
I started with ```agent_broken_connection_max_retry```, but then decided that was too long and went with ```broken_connection_max_retry```. It seems I forgot to change the doc string as well.